classes/services/queryBuilders/SubmissionListQueryBuilder.inc.php

```diff
@@ -44,6 +44,38 @@ public function filterBySections($sectionIds) {
 	 * @return object Query object
 	 */
 	public function appGet($q) {
+		$primaryLocale = \AppLocale::getPrimaryLocale();
+		$locale = \AppLocale::getLocale();
+
+		$this->columns[] = Capsule::raw('COALESCE(stl.setting_value, stpl.setting_value) AS section_title');
```

Comment:
We do lose much of the expressiveness of the query builder. But it does allow us to piece together bits of a query out of order, which is really helpful for building extensible queries to share code between OJS/OMP, and to allow plugins to hook in.

This method of gluing SQL strings together seems easier and less fragile to me than direct string manipulations. But I'd be open to an alternate approach. I just don't think I'm the one to formulate that.